servers/physics_3d/godot_collision_solver_3d_sat.cpp

```diff
@@ -860,7 +869,7 @@ static void _collision_sphere_box(const GodotShape3D *p_a, const Transform3D &p_
 	}
 	Vector3 point_a = p_transform_a.origin + (sphere_A->get_radius() + p_margin_a) * axis;
 	Vector3 point_b = (withMargin ? nearest + p_margin_b * axis : nearest);
```

Comment:
The `_collision_sphere_box` method has two issues:
 1. The axis points from A to B, where-as everywhere else it points from B to A.
 2. The "margins" are added in the same direction (the box margin actually pushes into the box)

If I correct these then then one of the regressions on https://github.com/fabriceci/Godot-Physics-Tests gets fixed - the one where the box tries to slide along the sphere.